spec/features/user_signs_in_spec.rb

```diff
@@ -14,15 +14,15 @@
     fill_in 'Password', with: user.password
 
     click_button 'Log in'
-
-    expect(page).to have_content('Signed in successfully')
+    visit users_path
+    expect(page).to have_content('Welcome')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.